dora/build.rs

```diff
@@ -34,10 +35,13 @@ fn main() -> Result<(), Box<dyn Error>> {
 
     writeln!(&mut stdlib, r#"["#,)?;
 
-    for f in fs::read_dir(SOURCE_DIR)? {
+    for f in WalkDir::new(SOURCE_DIR) {
         let f = f?;
 
-        if !f.file_type()?.is_file() {
+        if !f.file_type().is_file() {
+            if f.file_type().is_dir() {
+                create_dir_all(out_path.join(f.path()))?;
```

Comment:
Can we move this `create_dir_all` invocation, right before the copy call? Then we would only create directories with dora source files in them.